src/net_processing.cpp

```diff
@@ -1115,6 +1123,7 @@ PeerLogicValidation::PeerLogicValidation(CConnman* connmanIn, BanMan* banman, CS
 {
     // Initialize global variables that cannot be constructed at startup.
     recentRejects.reset(new CRollingBloomFilter(120000, 0.000001));
+    recent_confirmed_transactions.reset(new CRollingBloomFilter(24000, 0.000001));
```

Comment:
Add some comments to explain the parameter selection?

Reply:
Done.